framework/console/controllers/MessageController.php

```diff
@@ -521,6 +521,31 @@ private function extractMessagesFromTokens(array $tokens, array $translatorToken
     }
 
     /**
+     * @param string $category category that is checked
```

Comment:
Need a small explanation about how the ways how the method detects, whether the category is ignored